src/index.html

```diff
@@ -7,12 +7,55 @@
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="card">
+      <img
+        class="card__image"
+        src="./images/imac.jpeg"
+        alt="product_image"
```

Comment:
The 'alt' attribute should be descriptive. Instead of 'product_image', describe what is actually displayed in the image, for example, 'iMac with Retina 5K display'.